Readme.md

```diff
@@ -237,6 +237,85 @@ Your attributes will be automatically saved when you end the session, but if the
 
 We have wrapped up the above example into a high/low number guessing game skill you can [download here](https://github.com/alexa/skill-sample-nodejs-highlowgame).
 
+### Additional Event Processing
+
+The `OnRequest` event is similar to the `NewSession` event but is called regardless of the value of `event.session.new`. If you define `OnRequest` you cannot define `NewSession`. In fact all events will go to `OnRequest` and you will need to call another emit to have processing continue:
```

Comment:
Can you give me a use case on how you'd use this? In that use case would you want a catch-all specific to a particular state, or would you want the same logic to occur across multiple states (e.g., db initialization, logging, etc.)